spec/griddler/configuration_spec.rb

```diff
@@ -13,11 +13,12 @@
       expect(Griddler.configuration.processor_method).to eq(:process)
     end
 
-    it 'raises a helpful error if EmailProcessor is undefined' do
-      allow(Kernel).to receive_messages(const_defined?: false)
+    it 'raises a helpful error if EmailProcessor is undefined' # do
+    # allow(Kernel).to receive_messages(const_defined?: false)
```

Comment:
Could we do `stub_const(EmailProcessor, nil)` and check that the error is still raised?